File: src/main/java/me/creepinson/creepinoutils/api/util/data/game/IPlayer.java
package me.creepinson.creepinoutils.api.util.data.game;

import me.creepinson.creepinoutils.api.util.ISerializable;
import me.creepinson.creepinoutils.api.util.math.Tensor;

/**
 * @author Creepinson https:/theoparis.com/about
 **/
public interface IPlayer extends ISerializable {
	IWorld getWorld();

	Tensor getPosition();

	void setPosition(Tensor position);
}

File: src/main/java/dev/throwouterror/util/data/game/IPlayer.kt
package dev.throwouterror.util.data.game

import dev.throwouterror.util.ISerializable
import dev.throwouterror.util.math.Tensor

/**
 * @author Creepinson https:/theoparis.com/about
 */
interface IPlayer : ISerializable<Any?> {
    val world: IWorld?
    var position: Tensor?
}